src/test/java/com/timgroup/statsd/BlockingStatsDClientTest.java

```diff
@@ -0,0 +1,11 @@
+package com.timgroup.statsd;
+
+/**
+ * @author sfreitag
+ */
+public class BlockingStatsDClientTest extends StatsDClientTest<BlockingStatsDClient> {
+    @Override
+    public BlockingStatsDClient buildClient(String prefix, String[] constantTags) {
+        return new BlockingStatsDClient(prefix, STATSD_SERVER_HOST, STATSD_SERVER_PORT, constantTags);
+    }
+}
```

Comment:
Why would we need/want to? Wouldn't we want the spec for `NonBlocking` to be the same as `Blocking`?